src/docs/sphinx/dev_guide/updating_tpls.rst

```diff
@@ -0,0 +1,192 @@
+.. ## Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level LICENSE file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+.. _tpls-label:
+
+*********************
+Third-party Libraries
+*********************
+
+Axom dependencies are grouped into four categories: Git submodules,
+built-in TPLs in the Axom source tree, system-level TPLs, and other 
+TPL libraries. The following sections describe how to install and update 
+these dependencies for Axom.
+
+  * How does one add a new compiler or platform to the mix?
+  * How does one build a new set of TPLs for a single platform or compiler
+    for testing?
+  * What is the procedure for changing versions of one or more TPLs?
+  * How do we keep things straight when using different TPL versions for 
+    different branches?
+  * How to use the scripts for team TPL support vs. local development 
+    and experimentation?
+  * Others?
+
+
+=============
+Updating TPLs
+=============
+
+Git submodules
+--------------
+
+Currently, Axom uses three external packages that appear in the repo
```

Comment:
To avoid having to keep these docs consistent with the source repo, should we remove explicit lists like this and instead point users to the .gitmodules file to see which TPLs are managed as submodules?